ie_serving/config.py

```diff
@@ -16,12 +16,16 @@
 
 import os
 
-if os.path.isfile("/usr/local/lib/libcpu_extension.so"):
-    default_cpu_extension = "/usr/local/lib/libcpu_extension.so"
+LOCAL_LIB_CPU_EXTENSION = "/usr/local/lib/libcpu_extension.so"

```

Comment:
I would suggest to use a list of paths in a variable DEFAULT_CPU_EXTENSION_PATHS. It would include values depending on the package and distribution:
/usr/local/lib/libcpu_extension.so
inference_engine/lib/intel64/libcpu_extension_avx2.so
/usr/lib64/libcpu_extension.so
With that, dockerfile for clearlinux could be simplified - ENV dropped